bedrock/base/templates/includes/protocol/footer/firefox-new.html

```diff
@@ -0,0 +1,96 @@
+            <li><a href="{{ url('firefox.mobile') }}" data-link-type="footer" data-link-name="Mobile">{{ _('Mobile') }}</a></li>
+            <li><a href="https://mixedreality.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=reality" data-link-type="footer" data-link-name="Reality">{{ _('Reality') }}</a></li>
+            <li><a href="{{ url('firefox.enterprise.index') }}" data-link-type="footer" data-link-name="Enterprise">{{ _('Enterprise') }}</a></li>
+          </ul>
+        </section>
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            {{ _('Products') }}
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li><a href="https://lockwise.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=lockwise" data-link-type="footer" data-link-name="Lockwise">{{ _('Lockwise') }}</a></li>
+            <li><a href="https://monitor.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=monitor" data-link-type="footer" data-link-name="Monitor">{{ _('Monitor') }}</a></li>
+            <li><a href="https://send.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=send" data-link-type="footer" data-link-name="Send">{{ _('Send') }}</a></li>
+            <li><a href="#" data-link-type="footer" data-link-name="Browsers">{{ _('Browsers') }}</a></li>
+            <li><a href="https://getpocket.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=pocket" data-link-type="footer" data-link-name="Pocket">{{ _('Pocket') }}</a></li>
+          </ul>
+        </section>
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            {{ _('Join') }}
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li><a href="https://accounts.firefox.com/signup?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=signup" data-link-type="footer" data-link-name="Sign Up">{{ _('Sign Up') }}</a></li>
+            <li><a href="https://accounts.firefox.com/signin?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=signup" data-link-type="footer" data-link-name="Sign In">{{ _('Sign In') }}</a></li>
+            <li><a href="{{ url('firefox.accounts') }}" data-link-type="footer" data-link-name="Benefits">{{ _('Benefits') }}</a></li>
+          </ul>
+        </section>
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            {{ _('Developers') }}
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li><a href="{{ url('firefox.developer.index') }}" data-link-type="footer" data-link-name="Benefits">{{ _('Dev. Edition') }}</a></li>
```

Comment:
These all have `data-link-name="Benefits"` which I assume is copy-pasta, just noting it here.